### ISSUE TITLE
Commits and releases for language revisions

### ISSUE THREAD

**Issue by mgrojo** [reznikmm/adalib#1]
It would have been a good idea to have a different commit for each language version and a release for each one of them. Then the git history would match the language evolution.

At least could be done in this way for the next Ada version.

**Comment by reznikmm** [comment]
Yeah, it makes sense to have separate branches for each language revision.
